Sources/SimplenoteSearch/Extensions/NSPredicate+Search.swift

```diff
@@ -108,8 +108,15 @@ extension NSPredicate {
     /// Returns a NSPredicate that will match Tags with a given Keyword
     ///
     @objc
-    public static func predicateForTag(keyword: String) -> NSPredicate {
-        return predicateForTags(in: SearchQuery(searchText: keyword))
+    public static func predicateForTag(keyword: String, settings: SearchQuerySettings) -> NSPredicate {
+        return predicateForTags(in: SearchQuery(searchText: keyword, settings: settings))
+    }
+    
+    /// Returns a NSPredicate that will match Tags with a given SearchQuerty
+    ///
+    @objc
+    public static func predicateForTag(query: SearchQuery) -> NSPredicate {
```

Comment:
Let's drop this API, since it essentially only calls `predicateForTags(in` (and both are public).